com.unity.multiplayer.mlapi/Tests/Runtime/com.unity.multiplayer.mlapi.runtimetests.asmdef

```diff
@@ -1,7 +1,10 @@
 {
     "name": "Unity.Multiplayer.MLAPI.RuntimeTests",
+    "rootNamespace": "",
     "references": [
         "Unity.Multiplayer.MLAPI.Runtime",
+        "Unity.Multiplayer.NetworkProfiler.Runtime",
+        "Unity.Multiplayer.NetStats",
```

Comment:
This shouldn't go into develop since we're going to be an optional dependency. Is the intent that this will be removed before we merge? Or, more preferably, we should begin wrapping all the relevant code in the optional dependency ifdefs now.

Reply:
Yep, this will be fixed once we figure out the networkprofiler package and the relevant ifdef.

I have a wacky "ifdef true" everywhere where we need it right now.